mysql-ballerina/types.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/jballerina.java;
```

Comment:
Add missing license headers

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/104/commits/58551087f8f7b7517137fa8d16da9a7a24ae710c